CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 3.10.0
+
+- Add support for iOS promoted purchases (https://github.com/RevenueCat/purchases-flutter/pull/344)
+    New type: ReadyForPromotedProductPurchaseListener
+    New methods: addReadyForPromotedProductPurchaseListener(ReadyForPromotedProductPurchaseListener listener)
+                 removeReadyForPromotedProductPurchaseListener(ReadyForPromotedProductPurchaseListener listenerToRemove)
+  
+
```

Comment:
```suggestion
- Add support for iOS promoted purchases (https://github.com/RevenueCat/purchases-flutter/pull/344)
    New type: `ReadyForPromotedProductPurchaseListener`
    New methods:` addReadyForPromotedProductPurchaseListener(ReadyForPromotedProductPurchaseListener listener)`
                 `removeReadyForPromotedProductPurchaseListener(ReadyForPromotedProductPurchaseListener listenerToRemove)`
  
```

Reply:
By the way, those spaces and tabs are not respected on rendered markdown. You can see it in GitHub's rich diff:

<img width="990" alt="Screen Shot 2022-04-15 at 4 42 11 PM" src="https://user-images.githubusercontent.com/664544/163652951-64c0a7b0-8e18-470e-87e2-42fed3e96515.png">